fluentd/plan.sh

```diff
@@ -0,0 +1,47 @@
+pkg_name=fluentd
+pkg_origin=core
+pkg_version=0.14.0
+pkg_deps=(
+  core/bundler
+  core/ruby
+  core/gcc
+  core/make
+)
+pkg_build_deps=(
+  core/coreutils
+)
+pkg_source=http://nothing.to.download.from.com
```

Comment:
I think the current way of doing this is just leaving this line out, if there is no source to download. You should be able to remove this line completely.